tools/debugging/analyze_sp_logs.py

```diff
@@ -83,9 +83,12 @@ def write_translator(self) -> str:
 
 def detect_scenario_player_log(fn: os.PathLike) -> bool:
     with open(fn) as f:
-        for line in f.readlines():
-            if json.loads(line).get("logger") == "scenario_player.main":
-                return True
+        try:
+            for line in f.readlines():
+                if json.loads(line).get("logger") == "scenario_player.main":
+                    return True
+        except (UnicodeDecodeError, AttributeError):
```

Comment:
- `UnicodeDecodeError` is necessary for `.DSStore` files on macOS
- `AttributeError` is for non-json files